policyengine_uk/parameters/gov/dfe/study_childcare_entitlement/care_to_learn/amount.yaml

```diff
@@ -0,0 +1,24 @@
+description: The Department for Education provides this amount of financial support through the Care to Learn scheme.
```

Comment:
Not actually needed here (not a leaf node in the tree) but fine to keep in! No action needed.